### DISCUSSION TITLE
maxent script

### DISCUSSION THREAD

**Discussion by quantumable** [TRIQS/dft_tools#197] [General]
I did some DMFT calculations using dft_tools but I'm a bit stuck on the maxent analysis.

indeed i made the calculation for two correlated atoms which i want to plot each of the orbitals d(dxy,dxy,dz2,dxz,dx2-y2) for atom1 and 2. i used the model of the maxent script present here MAXENT.

i attach here the example of my maxent file. and the error message please do you have an idea for these two atoms? and when the calculation is spin polarized?

ERROR:

Traceback (most recent call last):
  File "max.py", line 40, in <module>
    gf = gf + G_latt['up'][iO,iO]
  File /TRIQS/install/lib/python3.8/site-packages/triqs/gf/gf.py", line 359, in __getitem__
    dat = self._data[ self._rank*(slice(0,None),) + key_tpl ]
IndexError: index 5 is out of bounds for axis 1 with size 5
[script.zip](https://github.com/TRIQS/dft_tools/files/7719683/script.zip)

**Comment by the-hampel** [comment]
This is a duplicate from https://github.com/TRIQS/maxent/issues/23 . Please report your issues only once. I will close the discussion here.
